CHANGE_LOG

```diff
@@ -1,3 +1,77 @@
+Version 0.45.0
+--------------
+
+In this release, Numba gained an experimental `numba.typed.List <pending_link>`_ container as a future replacement of the `reflected list <pending_link>`_. In addition, functions decorated with ``parallel=True`` can now be cached to reduce compilation overhead due to the auto-parallelization the function.
+
+
+Enhancements from user contributed PRs (with thanks!):
+
+James Bourbeau added Numba version in the error message in #4227, added the ``signature`` parameter to ``inspect_types`` in #4200, and
+improved the docstring of ``normalize_signature`` in #4205.
+
+Guilherme Leobas implemented the dominator tree and dominance frontier algorithms in #4216 and #4149
+
+Nick White fixed the issue with ``round`` in the CUDA target in #4137.
+
+Joshua Adelman added support for ``x in range(...)`` in #4129.
+
+
```

Comment:
Lucio Fernandez-Arjona added support for `np.select` in #4077 
Rob Ennis added support for `np.flatnonzero` in #4157

Reply:
i've updated the contributors